README.md

```diff
@@ -225,3 +225,82 @@ Example:
     ```
     pytest -s ecs/mxnet/training/test_ecs_mxnet_training.py::test_ecs_mxnet_training_dgl_cpu
     ```
+
+7. To run sagemaker Local tests, Launch a cpu or gpu instance based with latest deep learning AMI.
+    a) clone your github branch with changes and run the following commands
+       ```
+       git clone https://github.com/(github_account_id)/deep-learning-containers/
+       cd deep-learning-containers && git checkout (branch_name)
+       ```
+    b) login into the ecr repo where the new docker images build are present
+       ```
+       $(aws ecr get-login --no-include-email --registry-ids 669063966089 --region us-west-2)
+       ```
+    c) change to the sagemaker_tests/(framework)/(job_type) based the image to be tested
+       The example below refers to testing mxnet_training images
+       ```
+       cd test/sagemaker_tests/mxnet/traning/
+       pip3 install -r requirements.txt
+       ```
+    d) Run the sagemaker local tests using the below pytest command for images other than
+       tensorflow-inference
+       docker-base-name - ecr_url/repo name
+       tag - image_tag
+       py-version -  2 or 3 (image_major_python_version)
+       ```
+       python3 -m  pytest -v integration/local --region us-west-2 \
+       --docker-base-name (aws_account_id).dkr.ecr.us-west-2.amazonaws.com/beta-mxnet-inference \
```

Comment:
nit: for some reason I prefer {} around variable names, as it is more reminiscent of f"" strings in python, but will leave this up to you

Reply:
changed this